presto-iceberg/src/test/java/com/facebook/presto/iceberg/procedure/TestRemoveOrphanFilesProcedureHadoop.java

```diff
@@ -42,10 +43,10 @@ public TestRemoveOrphanFilesProcedureHadoop()
         super(HADOOP, ImmutableMap.of());
     }
 
-    @Override
+    @Test(dataProvider = "timezones")
```

Comment:
is this change intentional?

Reply:
Yes, seems we have to make this change after adding `@Test(singleThreaded = true)` on `TestRemoveOrphanFilesProcedureBase`, or we will get an exception as follows when running `TestRemoveOrphanFilesProcedureHadoop`:

```
Cannot inject @Test annotated Method [testRemoveOrphanFilesWithNonDefaultMetadataPath] with [class java.lang.String, boolean].
```